menas/src/main/scala/za/co/absa/enceladus/menas/models/rest/RestResponse.scala

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Wouldn't you rather extend the existing RestError class?

Reply:
1. Too afraid to touch Oozie stuff.
2. `RestError` is a case class. To extend it i'd probably need to replace it with a class hierarchy, which I'm not comfortable to do in Menas [yet].
3. Wanted for responses to have a certain structure, where `error` is an optional response field. 

But yes, ideas from `RestError` implementation were used to track issues by id and timestamp. 😄 

In the future, `RestError` and `RestResponse + ResponseError` might get combined (after refactoring) and a uniform approach is used.